src/components/structures/MessagePanel.js

```diff
@@ -372,11 +373,11 @@ export default class MessagePanel extends React.Component {
             return false; // ignored = no show (only happens if the ignore happens after an event was received)
         }
 
-        if (this._showHiddenEventsInTimeline) {
+        if (this.context?.showHiddenEventsInTimeline ?? this._showHiddenEventsInTimeline) {
```

Comment:
optional chaining in JS :(

Reply:
Also coalesce in JS

Reply:
Re: nullish coalescing, [same as above](https://github.com/babel/babel/pull/10811)